testing/cmake/write_version-relative/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ include(${rapids-cmake-dir}/cmake/write_version_file.cmake)
 
 cmake_minimum_required(VERSION 3.20)
 
-project(DEMO VERSION 3.2.1 LANGUAGES NONE)
+project(DEMO VERSION 3.2.0 LANGUAGES NONE)
```

Comment:
I think the only case not being tested is if this is 3.2, that the patch version would be 0